packages/itwin/map-layers/package.json

```diff
@@ -105,18 +101,18 @@
     "source-map-support": "^0.5.6",
     "ts-node": "^10.9.1",
     "typemoq": "^2.1.0",
-    "typescript": "~4.6.0"
+    "typescript": "~5.0.0"
   },
   "peerDependencies": {
     "@itwin/appui-abstract": "^4.0.0",
```

Comment:
maybe you should update the core peers to the latest?

Reply:
updated to latest